packages/openneuro-components/src/count-toggle/CountToggle.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+import { useQuery, useMutation } from 'graphql-hooks'
```

Comment:
These imports are available in @apollo/client and mixing them from another package will cause some subtle type mismatches.

```ts
import { useQuery, useMutation } from '@apollo/client'
```

Reply:
this was an unused import so I removed it.